kms_log/main.tf

```
@@ -186,6 +196,26 @@ resource "aws_cloudwatch_event_target" "sqs" {
     arn = "${aws_sqs_queue.kms_ct_events.arn}"
 }
 
+# event rule for custom events
+resource "aws_cloudwatch_event_rule" "unmatched" {
+    count = "${var.kmslogging_service_enabled}"
+    name = "${local.kmslog_event_rule_name}"
+    description = "Capture Unmatched KMS Log Events"
+
+    event_pattern = <<PATTERN
+{
+    "source":"gov.login.app"
```

```suggestion
    "source":["gov.login.app"]
```
Last change and then it works!